### ISSUE TITLE
How do we deploy: Use of Fluent / EF migrations and initial user?

### ISSUE THREAD

**Issue by mgalpin01** [mgalpin01/EM3#10]
Decision on how how we deploy (remove fluent / ef migrations if required) and intital user - one admin account change password on first login?

**Comment by sfackrell** [comment]
Some of this is now done.  Just need to edit ApplicationSeedData.cs for initial passwords.  If we want to force a user to choose a password on first login we'll need to add this as I don't think it's supported out of the box in ASP.NET Identity => would need a new bit flag and then some thought on how to get this to play nicely with the forms in Serenity.  Simpler quick alternative is to set new users a complicated password that they would want to change to something that they can remember, they can then do this manually once logged in or use forgot password/email functionality  if they can't get in

**Comment by mgalpin01** [comment]
Agreed on we don't bother with password expire on first logon - we'll just seed a single admin user 
Adminstrator role - tbc if this should have all permissions or just the Administration:Security (ability to add / remove permisisons), but or now we'll leave admin with everything
Also confirm if approach of removing role permissions in the role permission dialog is ok and no need to prevent removal eg of Administration:Security by directly accessing the endpoints is ok (although this won't be an issue if admin always has all permissions) 
Will keep this on hold until rest of permission model in place
